pkg/cloudprovider/providers/gce/gce_disks.go

```diff
@@ -675,22 +640,15 @@ func (gce *GCECloud) CreateDisk(
 
 	mc := newDiskMetricContextZonal("create", gce.region, zone)
 
-	createOp, err := gce.manager.CreateDiskOnCloudProvider(
+	err = gce.manager.CreateDiskOnCloudProvider(
 		name, sizeGb, tagsStr, diskType, zone)
 
 	if isGCEError(err, "alreadyExists") {
 		glog.Warningf("GCE PD %q already exists, reusing", name)
 		return nil
-	} else if err != nil {
+	} else {
 		return mc.Observe(err)
```

Comment:
do we still want to observe if err==nil? ditto elsewhere

Reply:
This was here before because it was an async operation.